.travis.yml

```diff
@@ -3,20 +3,20 @@ language: android
 android:
   components:
     - platform-tools
-    - build-tools-23.0.1
-    - android-23
+    - build-tools-23.0.3
+    - android-25
     - extra-android-m2repository
-    - sys-img-armeabi-v7a-android-18
+    - sys-img-armeabi-v7a-android-25
```

Comment:
The API 25 emulator won't work on Travis, but you're still creating an API 18 emulator below anyway so this should be kept at 18 as well.